examples/WriteText/WriteText.ino

```diff
@@ -0,0 +1,123 @@
+ * @version V1.0.0
+ * @date    6 July 2017
+ * @brief   Arduino test application for the STMicrolectronics
+ *          STM32L475V-DISCOVERY-IOT. NFC tag based on M24SR device.
+ *          This application makes use of C++ classes obtained from the C
+ *          components' drivers.
+ ******************************************************************************
+ * @attention
+ *
+ * <h2><center>&copy; COPYRIGHT(c) 2017 STMicroelectronics</center></h2>
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *   1. Redistributions of source code must retain the above copyright notice,
+ *      this list of conditions and the following disclaimer.
+ *   2. Redistributions in binary form must reproduce the above copyright notice,
+ *      this list of conditions and the following disclaimer in the documentation
+ *      and/or other materials provided with the distribution.
+ *   3. Neither the name of STMicroelectronics nor the names of its contributors
+ *      may be used to endorse or promote products derived from this software
+ *      without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ ******************************************************************************
+ */
+
+ /**
+ ******************************************************************************
+* How to use this sketch
+*
+* This sketch uses the I2C interface for communicate with the NFC device.
+* It writes an NFC tag type Text and reads this same tag.
+* Check the pinout and the I2C interface used for communication.
+* Choose the content of the text by changing the content of text_write.
+*
+* We can see on the monitor window the message "Sytstem init done!" when the
+* NFC module is started and ready. Next, the tag is written, we wait few seconds,
+* we read the same tag and print it on the monitor window.
+*
+* You can test this application by connecting it with your smartphone.
+* If you want to read tag in RF mode (with smartphone) unplug the USB on the board.
+* The board should be power-off, or RF mode will not be available.
+*
+* You can write a Text tag with your smartphone.
```

Comment:
Description not clear. In this case, the sample will write a text on the chip, then, it will read it to check if it is OK. With a smartphone, we will be able to check that text has been updated.
The read / write functionality from the smartphone is not dependent of the Arduino sketche...